### ISSUE TITLE
QUESTION: is it possible to ignore the schema id warning?

### ISSUE THREAD

**Issue by NoCopy** [ajv-validator/ajv#778]
Apologies if this has been covered, I couldn't find anything specifically covering this, but when using id vs $id is it possible to ignore / suppress the warning that states "schema id ignored XXXXXXX"

I am calling the schema from an Azure Cosmos DB, which has an "id" but is unrelated to the schema $id I want to validate. 

The issue is that each id / log warning goes into the Azure application insights, and frankly, is just cluttering the troubleshooting log; especially when the all errors config option is set.

Also additional apologies for not asking on https://gitter.im/ajv-validator/ajv according to the template but I can't understand why gitter requires access to basically my whole life and all my organizations data - recent Facebook hearings anyone?

**Comment by epoberezkin** [comment]
You should be able to manage it with options "schemaId" and "logger" - see https://github.com/epoberezkin/ajv#options

**Comment by NoCopy** [comment]
I did look at the options but as docs clearly state it will log warnings or throw exceptions on either of these items no matter what. Defining a specific logger only produces the warning in a different logger unless I would transform it.

> "$id" (default) - only use $id keyword as schema URI (as specified in JSON Schema draft-06/07), ignore id keyword (if it is present a warning will be logged).
> "id" - only use id keyword as schema URI (as specified in JSON Schema draft-04), ignore $id keyword (if it is present a warning will be logged).
> "auto" - use both $id and id keywords as schema URI. If both are present (in the same schema object) and different the exception will be thrown during schema compilation.

I am simply removing the id before validation now which I guess is the proper "valid" way of doing it anyways. Unfortunately, the DB requires "id" and it would be nice to suppress this particular as this is a common scenario; purely speculation but I am guessing that's one of the JSON Schema group made the change from id > $id in the first place.

That said, this is a fantastic library and quite frankly I wish I found it / used it some time ago. I do appreciate the hard work.
